authz-module/index.md

```diff
@@ -0,0 +1,204 @@
+```
+
+Install the `simd` binary:
+```
+make install
+```
+
+Check to make sure the installation was successful:
+```
+simd version
+```
+
+You should see `0.43.0` printed to the console.
+
+## Configuration
+
+<!-- TODO: update tip to use `unsafe-reset-all` with better user experience -->
+
+::: tip
+If you have used `simd` before, you might already have a `.simapp` directory in your home directory. To keep the previous data, either save the directory to another location or use the `--home` flag and specify a different directory for each command in the following instructions. If you don't want to keep the previous data, remove the previous directory (`rm -rf ~/.simapp`).
+:::
+
+Run the following commands to configure the `simd` binary.
+
+Set the chain ID:
+```
+simd config chain-id demo
+```
+
+Set the keyring backend:
+```
+simd config keyring-backend test
+```
+
+## Key Setup
+
+Create a couple of test keys. Use the `--recover` option so that the addresses used in the following example commands are consistent.
+
+Add a key for Alice:
+```
+simd keys add alice --recover
+```
+
+Enter the following mnemonic:
+```
+plunge hundred health electric victory foil marine elite shiver tonight away verify vacuum giant pencil ocean nest pledge okay endless try spirit special start
+```
+
+Add a key for Bob:
+```
+simd keys add bob --recover
+```
+
+Enter the following mnemonic:
+```
+shuffle oppose diagram wire rubber apart blame entire thought firm carry swim old head police panther lyrics road must silly sting dirt hard organ
+```
+
+## Chain Setup
+
+The following commands start a chain for the simulation application (`simapp`).
+
+Initialize the node:
+```
+simd init test --chain-id demo
+```
+
+Add Alice and an initial balance to the genesis file:
+```
+simd add-genesis-account alice 5000000000stake
+```
+
+Add Bob and an initial balance to the genesis file:
+```
+simd add-genesis-account bob 5000000000stake
+```
+
+Generate a transaction adding Alice to the initial validator set:
+```
+simd gentx alice 1000000stake --chain-id demo
+```
+
+Add the validator transaction to the genesis file:
+```
+simd collect-gentxs
+```
+
+## Start Chain
+
+You are now ready to start a single node network on your local machine.
+
+Start the chain:
+```
+simd start
+```
+
+## Submit Proposal
+
+To demonstrate an authorization to vote on a governance proposal, you must first create a governance proposal.
+
+Create proposal:
+```
+simd tx gov submit-proposal --title="Test Authorization" --description="Is Bob authorized to vote?" --type="Text" --deposit="10000000stake" --from alice
+```
+
+View proposal:
+```
+simd q gov proposal 1
+```
+
+## Grant Authorization
+
+Next, the granter must `grant` an authorization to the grantee.
+
+The authorization is a "generic" authorization, meaning there are no limits on the authorization. In this case, the grantee is able to vote as many times as they want until the granter revokes the authorization.
```

Comment:
Maybe we could add a link to an overview of other authorization types besides "generic"? Not sure if that exist?

Reply:
I wonder the same thing! IIUC, the authorization levels are full, restricted, and hidden. 
```suggestion
The authorization is a full authorization, which means there are no limits on the authorization. In this case, the grantee is able to vote as many times as they want until the granter revokes the authorization.
```

Reply:
The "generic" authorization is one of five supported authorization types.

```
Usage:
  simd tx authz grant <grantee> <authorization_type="send"|"generic"|"delegate"|"unbond"|"redelegate"> --from <granter> [flags]
```

I will make this more clear and point to the documentation that provides more information:
https://docs.cosmos.network/master/modules/authz/01_concepts.html#built-in-authorizations

Reply:
today I learned about authorization types, yes, link to that section :)